exec/csnex/Global.cs

```diff
@@ -464,11 +484,209 @@ public void number__toString()
         }
 #endregion
 #region Object Functions
+        #region Object Getters
+        public void object__getArray()
+        {
+            Object obj = Exec.stack.Pop().Object;
+
+            List<Object> a = new List<Object>();
```

Comment:
There's no purpose in creating a new `List` here, only to throw it away in the next call. Either don't initialize this reference (if the compiler lets you) or initialize it to null.

Reply:
Fixed.